nr1.json

```diff
@@ -1,6 +1,6 @@
 {
   "schemaType": "NERDPACK",
-  "id": "4da92288-f951-41c5-a91c-9e45d4864ee9",
+  "id": "8868862e-1784-4e3b-9101-b02beea955ad",
```

Comment:
what caused the ID to change? just curious.

Reply:
Oh that was just me regenerating the uuid 🗡️ It won't matter, the catalog will ignore this anyway